packages/bits/src/lib/toolbar/toolbar.component.html

```diff
@@ -1,44 +1,92 @@
-<div class="nui-toolbar-content d-flex flex-row align-items-center justify-content-between"
-     nuiResizeObserver
-     (containerResize)="moveToolbarItems()"
-     #toolbarContainer>
-    <div class="nui-toolbar-content__dynamic d-flex flex-row flex-shrink-0">
-        <ng-template ngFor let-i="index" let-commandGroup let-last="last" [ngForOf]="commandGroups">
-            <ng-template ngFor let-commandItem let-j="index" [ngForOf]="commandGroup.items">
+<div
+    class="nui-toolbar-content"
+    nuiResizeObserver
+    (containerResize)="moveToolbarItems()"
+    #toolbarContainer
+>
+    <div class="nui-toolbar-content__dynamic">
+        <ng-template
+            ngFor
+            let-i="index"
+            let-commandGroup
+            let-last="last"
+            [ngForOf]="commandGroups"
+        >
+            <ng-template
+                ngFor
+                let-commandItem
+                let-j="index"
+                [ngForOf]="commandGroup.items"
+            >
                 <!-- (!i && !j) is used for set focusable state to first element first group -->
-                <button nui-button type="button"
-                        #toolbarButtons
-                        [tabindex]="(!i && !j) ? 0 : -1"
-                        (click)="onClickToolbarBtn($event, commandItem)"
-                        [displayStyle]="commandItem.displayStyle === 'main' ? 'primary' : commandItem.displayStyle"
-                        [ngClass]="{'nui-toolbar-primary-item': commandItem.displayStyle === 'main'}"
-                        [icon]="commandItem.icon">
-                    <span>{{commandItem.title}}</span>
+                <button
+                    nui-button
+                    type="button"
+                    #toolbarButtons
+                    [tabindex]="!i && !j ? 0 : -1"
+                    (click)="onClickToolbarBtn($event, commandItem)"
+                    [displayStyle]="
+                        commandItem.displayStyle === 'main'
+                            ? 'primary'
+                            : commandItem.displayStyle
```

Comment:
tbh I am not a big fan of this formatting in the templates, I think it's much clearer if it stays one attribute per line, even if the line is a slightly longer.